openstates/me/people.py

```diff
@@ -119,30 +143,30 @@ def scrape_reps(self):
 
                     person.add_contact_detail(
                         type='address', value=leg_address, note='Party Office')
-
                 elif party == 'Republican':
                     leg_address = (
                         'House GOP Office, Room 332 State House, 2 State House Station, '
                         'Augusta, Maine 04333-0002'
                     )
-
                     person.add_contact_detail(
                         type='address', value=leg_address, note='Party Office')
 
             if office_email:
-                office_email = office_email[0]
                 person.add_contact_detail(type='email', value=office_email, note='District Office')
```

Comment:
Seems like these are `Capitol Office` email addresses. they all end in `legislature.maine.gov`.